assets/src/dashboard/app/views/shared/telemetryBanner.js

```diff
@@ -142,7 +153,18 @@ export const TelemetryOptInBanner = forwardRef(
           </ToggleButton>
         </Header>
         <Label>
-          <CheckBox checked={checked} disabled={disabled} onChange={onChange} />
+          <CheckBox
+            checked={checked}
+            disabled={disabled}
+            onChange={() => {
+              onChange();
+              focusOnCheckbox.current = true;
+            }}
+            onBlur={() => {
+              focusOnCheckbox.current = false;
+            }}
```

Comment:
It seems the every time the checkbox is checked/unchecked, the entire component re-renders and thus focus is lost.  I can't think of any quick/clean way to prevent this so nice workaround to keep the focus!